pai-management/doc/how-to-write-pai-configuration.md

```diff
@@ -185,7 +185,7 @@ cluster:
     - ```docker-registry-domain```: E.g., gcr.io. If public，fill docker_registry_domain with the word "public".
     - ```docker-username```: The account of the docker registry
     - ```docker-password```: The password of the account
-    - ```docker-tag```: The image tag of the service. You could set the version here. Or just set latest here.
+    - ```docker-tag```: The image tag of the service, which should match the PAI release version, for example: `v0.6.1` for the release 0.6. Or use `latest` for the latest code.
```

Comment:
0.6 —> 0.6.1;
the latest code —> the latest code in master branch.

Reply:
Yes, you noticed the difference.
Here's something I want to express: when user choose release 0.6 (so emit the third version), I want to recommend the latest tag(that's why it include the third).